src/Centreon/Domain/HostConfiguration/UseCase/V21/FindHostTemplates.php

```diff
@@ -50,12 +51,14 @@ public function __construct(HostConfigurationReadRepositoryInterface $configurat
      * Execute the use case for which this class was designed.
      *
      * @return FindHostTemplatesResponse
-     * @throws \Exception
+     * @throws \Throwable
```

Comment:
Even if `RepositoryException` is included in `Throwable`, should you add it to phpdoc ?